jupyter/src/incremental_learning/job.py

```diff
@@ -200,6 +200,30 @@ def get_predictions(self) -> np.array:
                                    ['ml']['{}_prediction'.format(self.dependent_variable)])
         return np.array(predictions)
 
+    def get_probabilities(self) -> Union[None, np.array]:
+        """Return prediction probabilities for the class "true".
+
+        Note: Only works for binary classification.
+
+        Returns:
+            Union[None, np.array]: prediction probabilities. 
+        """
+        if self.analysis_name == 'regression':
```

Comment:
Could we cache the number of classes in the `self` and fail as well if > 2. It means it won't fail silently and just output nonsense. Alternatively, we could add output top classes to the job configs and get from there (this will work in all cases). This can be done automatically by the framework, i.e. setting from `num_classes` in the config file.

Reply:
Good point, now I raise exceptions if it's `regerssion` or `num_classes` !=2 .